pkg/aws/policies.go

```diff
@@ -373,9 +373,6 @@ func (c *awsClient) HasCompatibleVersionTags(iamTags []*iam.Tag, version string)
 	}
 	for _, tag := range iamTags {
 		if aws.StringValue(tag.Key) == tags.OpenShiftVersion {
-			if version == aws.StringValue(tag.Value) {
-				return true, nil
-			}
```

Comment:
This will recreate policies every time you create a cluster which isn't the intention of have a set version policies. The intention is to send the least amount of IAM calls to AWS.

Reply:
Yes, I had setup a check for each action before. However, @pvasant considered that approach overkill

Reply:
Maybe we could implement a new flag to allow a user to force new policies?

Reply:
Sure!